app/views/match_mailer/match_date_changed.text.erb

```diff
@@ -1,7 +1,18 @@
 <%= t('mailer.hello', firstname: @user.first_name) %>
 
-<%= t('match_mailer.next_game', name: @match.event.name, team_home: @match.team_home.name, team_away: @match.team_away.name) %>
-<%= t('.info', start_time: I18n.localize(@match.start_time)) %>
+<%= t('.info_txt',
+      team_name: @opponent_team.name,
+      event_name: @match.event.name) %>
 
-<%= t('mailer.have_fun') %>
-<%= event_overview_url(@match.event) %>
\ No newline at end of file
+<% if @match.start_time %>
+<%= t('match_mailer.start_time',
+      start_date: I18n.localize(@match.start_time.to_date),
+      start_time: @match.start_time.to_s(:time)) %>
+<% end %>
+<% if @match.place %>
+<%= t('match_mailer.place', place: @match.place) %>) %><
```

Comment:
You accidentally added a `<` at the end of this line

Reply:
This results in the following rendering:

<img width="532" alt="bildschirmfoto 2018-02-06 um 22 57 15" src="https://user-images.githubusercontent.com/807665/35886373-1d5c0776-0b91-11e8-9724-8b251c768485.png">

Reply:
**The same applies for all the other templates including the match_mailer.place tag!**